alma/controllers/front/payment.php

```diff
@@ -133,6 +143,12 @@ public function postProcess()
             return;
         }
 
+        if (Settings::activateFragment()) {
+            method_exists(get_parent_class($this), 'ajaxDie')
```

Comment:
@Francois-Gomis 
I don't know, it was already like this. I suppose it was for old version of Prestashop

Reply:
`    /**
     * @deprecated deprecated since 1.7.5.0, use ajaxRender instead
     * Dies and echoes output value
     *
     * @param string|null $value
     * @param string|null $controller
     * @param string|null $method
     *
     * @throws PrestaShopException
     */
    protected function ajaxDie($value = null, $controller = null, $method = null)
    {
        $this->ajaxRender($value, $controller, $method);
        exit;
    }
`

Reply:
https://github.com/PrestaShop/PrestaShop/blob/develop/classes/controller/Controller.php

Reply:
First AjaxDie() in V1.6.0.12 : 
https://github.com/PrestaShop/PrestaShop/blob/1.6.0.12/classes/controller/Controller.php

Reply:
Can you create a method with the good method with version comapre, or with method_exits with version comment ( upper than 1.7.5.0 - between 1.6.0.12 and 1.7.5.0 - before 1.6..0.12 ?

Reply:
👍